source/_patterns/02-molecules/vue-widgets/index.js

```diff
@@ -12,10 +12,11 @@ import './_vue-facet-table-vuex.twig';
 import './_vue-facet-table-vue.twig';
 
 // Module styles
-import './_example-widget.scss';
+import './_vue-widgets.scss';
 
-import FacetTableVue from './src/vue-example'; // mount only if dom found
-import FacetTableVueX from './src/vuex-example'; // mount only if dom found
+import FacetTableVue from './src/vue-cryptos'; // load only if dom found
```

Comment:
No need for the inline comments per line here. If anything, put it in a block above (that would be nice actually)